pywr/parameters/_parameters.pyx

```diff
@@ -1377,6 +1377,56 @@ cdef class DeficitParameter(Parameter):
 DeficitParameter.register()
 
 
+cdef class FlowParameter(Parameter):
+    """Parameter that provides the flow from a node from the previous time-step.
```

Comment:
What is the value of this parameter on the initial timestep? Does it need an initial value?

Reply:
Good point. It is currently zero and not configurable.